cli/script/command-executor.ts

```diff
@@ -557,6 +571,7 @@ function login(command: cli.ILoginCommand): Promise<void> {
 
 function loginWithExternalAuthentication(action: string, serverUrl?: string): Promise<void> {
     initiateExternalAuthenticationAsync(action, serverUrl);
+    log("");    // Insert newline
```

Comment:
Because I removed '/r/n' from `initiateExternalAuthentication...()`, which is used by `link`